config/generated.go

```diff
@@ -1,11 +1,11 @@
 package config
```

Comment:
don't touch this file, as it is generated during build, no need to commit this file please.